Sources/Swift/Protocol/SentryLogAttribute.swift

```diff
@@ -1,91 +1,121 @@
 extension SentryLog {
-    enum Attribute: Codable {
-        case string(String)
-        case boolean(Bool)
-        case integer(Int)
-        case double(Double)
+    @objc(SentryStructuredLogAttribute)
+    @objcMembers
+    public class Attribute: NSObject, Codable {
+        public let type: String
```

Comment:
`m`: Not sure on the impact, but should we offer static convenience methods for `string(..)`, `boolean(..)`, `integer(..)` and `.double(..)` so it's not a breaking change?

Reply:
This was internal API before, so not breaking.